index.js

```diff
@@ -16,3 +18,18 @@ mobileLinks.forEach((link) => {
     mobileMenu.classList.toggle('active');
   });
 });
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    if (email.validity) {
+      email.setCustomValidity('Email must be in lowercase');
+    }
+    e.preventDefault();
+  }
+});
+
+email.addEventListener('input', () => {
+  if (email.validity) {
+    email.setCustomValidity('');
+  }
+});
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/9_validate_form.md#:~:text=If%20the%20validation%20is%20not%20OK%2C%20you%20show%20an%20error%20message%20to%20the%20user%20near%20the%20submit%20button%20informing%20them%20of%20the%20error%20and%20the%20form%20is%20not%20sent.), Kindy display the error message near the submit button.

https://user-images.githubusercontent.com/68796232/231215785-40dc4043-9623-4f9a-8684-af997c509ea9.mp4